README.md

```diff
@@ -724,6 +727,90 @@ KlaviyoSDK().unregisterFromInAppForms()
 
 Note that after unregistering, the next call to `registerForInAppForms()` will be considered a new session by the SDK.
 
+## Geofencing
+
+[Geofencing](https://help.klaviyo.com/hc/en-us/articles/360023213971) allows you to trigger events when users enter or exit geographic regions. The Klaviyo SDK monitors geofences configured in your Klaviyo account and automatically tracks geofence enter and exit events.
```

Comment:
just leaving a note to update this once available.

Reply:
Minor: why not just leave this unlinked, rather than risk forgetting to update it and have it linking to the wrong thing?